File: Source/Addins/Eto.Addin.XamarinStudio/Templates/Files/VB/code/dialog.vb
using System;
using System.Collections.Generic;
using Eto.Forms;
using Eto.Drawing;

namespace ${Namespace}
{	
	public class ${EscapedIdentifier} : Dialog
	{	
		public ${EscapedIdentifier}()
		{
			Title = "My ${EscapedIdentifier} dialog";

			// buttons
			DefaultButton = new Button { Text = "OK" };

			AbortButton = new Button { Text = "C&ancel" };
			AbortButton.Click += (sender, e) => Close();

			var buttons = new TableLayout { Rows = { new TableRow(null, DefaultButton, AbortButton) }, Spacing = new Size(5, 5) };

			Content = new TableLayout
			{
				Padding = new Padding(10),
				Rows = 
				{
					new Label { Text = "Some content" },
					buttons
				}
			};
		}
	}
}

File: src/Addins/Eto.Addin.MonoDevelop/Templates/Files/FS/code/dialog.fs
namespace ${Namespace}

open System
open Eto.Forms
open Eto.Drawing

type ${EscapedIdentifier}Base () as this =
    inherit Dialog ()

    do
        base.Title <- "My ${EscapedIdentifier} dialog"
        base.Content <- new Label (Text = "Some Content")
